### PR TITLE
Removed Dead code

### PR DESCRIPTION
**Pull request by tockards**
Merging @amakhaba branch into initial branch of the py2 to py3 migration. 

*Definition of Done Checklist*

- [x] Code meets our [python style guidelines](https://docs.google.com/document/d/1aZoIyR9tz5rCWr2qJKuMTmKp2IzHlFjrCFrpDDHFypM/edit?usp=sharing)?
- [x] Unit tested (coded, passed, included)?
- [x] Requested at least 2 reviewers?
- [x] Commented code, particularly in hard-to-understand areas?
- [x] Made corresponding changes to the documentation (e.g. Python documentation, System Engineering Documentation, version description updates, README file, etc)?

JIRA: [MT-738](https://skaafrica.atlassian.net/browse/MT-738)

### CODE REVIEW

File: tango_simlib/utilities/sim_xmi_parser.py [hunk 1]
```diff
@@ -173,17 +173,7 @@ def extract_device_class_descr(self, description_data):
...
-                                                # class, however it is not useful for
-                                                # the current problem.
-        #class_data['identification'] = {}
-        #identification = description_data.find('identification')
-        #class_data['identification']['contact'] = identification.attrib['contact']
-        #class_data['identification']['author'] = identification.attrib['author']
-        #class_data['identification']['emailDomain'] = (
-        #    identification.attrib['emailDomain'])
-        #    class_data['identification'].append(id.attrib)
+
```
Comment (mmphego) [inline tango_simlib/utilities/sim_xmi_parser.py]: This change is already in `master`.
See: https://github.com/ska-sa/tango-simlib/pull/103/files

**Comment by tockards** [comment]
@mmphego 
this PR addresses 
>Merging @amakhaba branch into initial branch of the py2 to py3 migration.

Should I rather branch off master? I am currently using the previous py2topy3 branch as I worked with it yesterday and I am continuing with getting the tox file and coverage to use enable the nose-xunitmp settings.

**Comment by mmphego** [comment]
> @mmphego
> this PR addresses
> 
> > Merging @amakhaba branch into initial branch of the py2 to py3 migration.
> 
> Should I rather branch off master? I am currently using the previous py2topy3 branch as I worked with it yesterday and I am continuing with getting the tox file and coverage to use enable the nose-xunitmp settings.

@amakhaba and I are currently preparing the branch for futurize stage 1. See PR#105. 
Can you please look into futurize stage 2 PY2 tests, as soon as that PR is merged we will do stage 1 -> black and then tackle stage 2.

We are trying to follow the wiki as we update and improve it

**Comment by tockards** [comment]
sure and thanks. I will close this PR so long.

**Pull request closed without merging**